### DISCUSSION TITLE
Future Support for Existing Screenshot Testing Tools

### DISCUSSION THREAD

**Discussion by toucheGuorin** [stenciljs/core#5846] [Stencil Testing]
Hi Stencil team,

I spotted this [comment about the deprecation of Stencil screenshot testing](https://github.com/ionic-team/stencil/issues/5563#issuecomment-2176675773). I was wondering if, going forward, both Playwright and WebdriverIO integrations with Stencil will be supported in the long-term, or if there is a chance for either to be deprecated similarly to Stencil screenshot testing?

Thanks

**Comment by christian-bromann** [comment]
> I was wondering if, going forward, both Playwright and WebdriverIO integrations with Stencil will be supported in the long-term

Our current goal is to slim down Stencil to its core value: building web components and remove everything that other tooling may do better, e.g. testing. As oppose to Stencil's snapshot testing which never went out of experimental state, we have released support for WebdriverIO and Playwright to enable Stencil developers to test their components effectively. We obviously have no control about the longevity of these projects but especially tools like WebdriverIO are build for long-term use and reliability.
